data/dev/analysis/ota/ota002/rubrics/noise_white.json

```diff
@@ -0,0 +1,179 @@
+{
+  "rubric_id": "noise_white",
+  "version": "0.1.0",
+  "weights": {
+    "correctness": 0.4,
+    "completeness": 0.2,
+    "groundedness": 0.2,
+    "device_specificity": 0.1,
+    "safety": 0.1
+  },
+  "criteria": [
+    {
+      "id": "topology",
+      "desc": "Explicitly identifies topology in a Topology section",
+      "required": true,
+      "section": "Topology",
+      "patterns_any": [
+        "telescopic",
+        "cascode",
+        "cascoded",
+        "differential",
+        "fully differential"
+      ],
+      "min_any": 2,
+      "anti_patterns": [
+        "single-ended",
+        "two-stage",
+        "folded"
+      ],
+      "weight": 0.1
+    },
+    {
+      "id": "key_relation",
+      "desc": "States output-referred thermal noise CURRENT: i\u00b2n,out = (8kT/3)(gm_M1 + gm_M2 + gm_M7 + gm_M8). Input pair (M1, M2) + current mirror (M7, M8). Cascodes M3-M6 don't contribute (noise circulates)",
+      "required": true,
+      "patterns_any": [
+        "i\u00b2n,out = (8kT/3)(gm_M1 + gm_M2 + gm_M7 + gm_M8)",
+        "i\u00b2n = 4kT\u03b3(gm_M1 + gm_M2 + gm_M7 + gm_M8)",
+        "(gm_M1 + gm_M2 + gm_M7 + gm_M8)",
+        "gm1 + gm2 + gm7 + gm8"
```

Comment:
Need to multiplex somehow for Cascode modality. Top CurrentMirrorPMOS motif contributes noise and InputPairNMOS motif contributes noise. I think it's ok to put this on the backlog for now since the rubric refactor to use Markdown will directly follow our two merges.

@KesavaViswanadha can you make an issue that tracks this problem? "Analysis rubric tests device identification which is not valid for Cascode modality" - so we don't lose track